lesson02/assignment/css/_components.content.scss

```diff
@@ -1,17 +1,117 @@
-html {
-    box-sizing: border-box;
+/*style for outter gray box*/
+.main {
+    width: 951px;
+    height: 474px;
+    background-color: $outter-gray; /*variable*/
+    border: 0.5px solid #d3d3d3;
+    border-radius: 5px;
```

Comment:
Border width and border radius are the two main exceptions to the "don't use px for sizing" rule. This is just fine, no need to change.